setup.cfg

```diff
@@ -5,7 +5,7 @@ author_email = holger@merlinux.eu
 license = MIT
 license_files = LICENSE
 url = https://github.com/pytest-dev/pytest-xprocess/
-version = 0.13.1
+version = 0.14.0
```

Comment:
Ahh we can remove this line completely, the version is set by `setuptools_scm`. 👍

Reply:
Oh, missed that one. Thanks!